2021-01-24 07:04:36,723 INFO    MainThread:5731 [wandb_setup.py:_flush():70] setting env: {}
2021-01-24 07:04:36,723 INFO    MainThread:5731 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-24 07:04:36,724 INFO    MainThread:5731 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-24 07:04:36,724 INFO    MainThread:5731 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-24 07:04:36,724 INFO    MainThread:5731 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/4doa/wandb/run-20210124_070436-hfvp5k97/logs/debug.log
2021-01-24 07:04:36,724 INFO    MainThread:5731 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/4doa/wandb/run-20210124_070436-hfvp5k97/logs/debug-internal.log
2021-01-24 07:04:39,295 INFO    MainThread:5731 [wandb_run.py:_console_start():1308] atexit reg
2021-01-24 07:04:39,295 INFO    MainThread:5731 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-24 07:04:39,296 INFO    MainThread:5731 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-24 07:04:39,296 INFO    MainThread:5731 [redirect.py:install():205] install start
2021-01-24 07:04:39,296 INFO    MainThread:5731 [redirect.py:install():220] install stop
2021-01-24 07:04:39,296 INFO    MainThread:5731 [redirect.py:install():205] install start
2021-01-24 07:04:39,297 INFO    MainThread:5731 [redirect.py:install():220] install stop
2021-01-24 07:04:39,297 INFO    MainThread:5731 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-24 07:04:52,717 INFO    MainThread:5731 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-24 07:04:52,718 INFO    MainThread:5731 [wandb_run.py:_restore():1250] restore
2021-01-24 07:04:52,718 INFO    MainThread:5731 [redirect.py:uninstall():224] uninstall start
2021-01-24 07:04:52,718 INFO    MainThread:5731 [redirect.py:_stop():279] _stop: stdout
2021-01-24 07:04:52,718 INFO    stdout    :5731 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-24 07:04:52,718 INFO    stdout    :5731 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:_stop():285] _stop closed: stdout
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:_stop():291] _stop joined: stdout
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:uninstall():228] uninstall done
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:uninstall():224] uninstall start
2021-01-24 07:04:52,719 INFO    MainThread:5731 [redirect.py:_stop():279] _stop: stderr
2021-01-24 07:04:52,720 INFO    stderr    :5731 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-24 07:04:52,720 INFO    stderr    :5731 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-24 07:04:52,720 INFO    MainThread:5731 [redirect.py:_stop():285] _stop closed: stderr
2021-01-24 07:04:52,720 INFO    MainThread:5731 [redirect.py:_stop():291] _stop joined: stderr
2021-01-24 07:04:52,720 INFO    MainThread:5731 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-24 07:04:52,720 INFO    MainThread:5731 [redirect.py:uninstall():228] uninstall done
2021-01-24 07:04:53,052 INFO    MainThread:5731 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "13af720e17944e0d9681f01fd2afec0c"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-01-24 07:04:55,055 INFO    MainThread:5731 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "d1b5eaeb8eef48adb21a69843598e4eb"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5182
      total_bytes: 5182
    }
  }
}

2021-01-24 07:04:56,273 INFO    MainThread:5731 [wandb_run.py:_show_files():1614] logging synced files
2021-01-24 07:04:56,274 INFO    MainThread:5731 [internal.py:handle_exit():139] Internal process exited
